### PR TITLE
Add BDN tests:  Operations.RawStringOperations and Operations.CustomOperations

### PR DESCRIPTION
**Pull request by darrenge**
This PR does the following:
1) Adds BDN tests:   Operations.RawStringOperations and Operations.CustomOperations and the corresponding expected values in the config file. Required a minor change in the .ps1 file for the results array (change from size 20 to 70)
2) Adds a new badge (BDN Charts) on to the front of Github.com page (https://github.com/microsoft/garnet)
3) Adds a link to the charts under the "Benchmarking" category on the left (https://microsoft.github.io/garnet/docs/benchmarking/overview)

### CODE REVIEW

**Review by badrishc — APPROVED** [review]

File: .github/workflows/ci-bdnbenchmark.yml [hunk 1]
```diff
@@ -42,7 +42,7 @@ jobs:
         os: [ ubuntu-latest, windows-latest ]
         framework: [ 'net8.0' ]
         configuration: [ 'Release' ]
-        test: [ 'Operations.BasicOperations', 'Operations.ObjectOperations', 'Cluster.ClusterMigrate', 'Cluster.ClusterOperations', 'Lua.LuaScripts' ]
+        test: [ 'Operations.BasicOperations', 'Operations.ObjectOperations', 'Cluster.ClusterMigrate', 'Cluster.ClusterOperations', 'Lua.LuaScripts', 'Operations.CustomOperations','Operations.RawStringOperations' ]
```
Comment (badrishc) [inline .github/workflows/ci-bdnbenchmark.yml:45]: Add [ScriptOperations](https://github.com/microsoft/garnet/blob/09ef77042c08680ca4e76ce78f0c9c72b324c1b1/benchmark/BDN.benchmark/Operations/ScriptOperations.cs) as well.

Comment (darrenge) [reply .github/workflows/ci-bdnbenchmark.yml]: Done!

File: .github/workflows/ci-bdnbenchmark.yml [hunk 2]
```diff
@@ -42,7 +42,7 @@ jobs:
         os: [ ubuntu-latest, windows-latest ]
         framework: [ 'net8.0' ]
         configuration: [ 'Release' ]
-        test: [ 'Operations.BasicOperations', 'Operations.ObjectOperations', 'Cluster.ClusterMigrate', 'Cluster.ClusterOperations', 'Lua.LuaScripts' ]
+        test: [ 'Operations.BasicOperations', 'Operations.ObjectOperations', 'Cluster.ClusterMigrate', 'Cluster.ClusterOperations', 'Lua.LuaScripts', 'Operations.CustomOperations','Operations.RawStringOperations','Operations.ScriptOperations' ]
```
Comment (badrishc) [inline .github/workflows/ci-bdnbenchmark.yml:45]: There was another merge to main that adds `HashObjectOperations`, so we need to add this to the PR as well. Hopefully this is the last and we can merge after it.

Comment (darrenge) [reply .github/workflows/ci-bdnbenchmark.yml]: Yep ... merged

**Pull request merged**